### DISCUSSION TITLE
2023-08-16

### DISCUSSION THREAD

**Discussion by nlom0218** [woowacourse-teams/2023-haru-study#350] [daily_meeting]
### 개인 일정

- 마코, 룩소: 4시반 근로
- 노아: 5시 근로

### 오늘의 할 일

- 각 분야끼리 지금까지 한 일 동기화
- 체크리스트
- 글쓰기 미션(모디)

#### 백엔드

- 리뷰후 merge

#### 프론트엔드

- 오늘 중으로 새로운 api 적용 작업
- 개인 기록 페이지
- 프로필 뱃지(with 히이로)

### 내일의 할 일

- 체크리스트

### 언젠가 할 일

- 소켓, 폴링 양방향 통신 어떻게 할지 (4차 스프린트 내에 그냥 진행하고 발표만 안하는 방향으로 갈건지 로그인 이후 결정하기)
- 질문 PlaceHolder로 유의미한 응답 예시를 제공하는 기능 (목표설정 가이드라인)
- 브랜치 전략 결정하기(dev -> main)
- A라는 스터디를 참여후 B스터디 참여한 뒤 다시 A스터디에 참여할 시 닉네임을 찾지 못한 이슈(로그인이 된다면 해결될 문제이긴 함)

**Comment by aak2075** [comment]
### 체크인
10점 - 잠도 잘 자고 컨디션 좋아요

### 체크아웃
10점 - 무난합니다

**Comment by jaehee329** [comment]
### 체크인
8점: 빨리 방학했으면 좋겠어요~~ㅠㅠㅠㅠ

### 체크아웃
7점: 오늘 맛있는게 없어서 아쉬워요

**Comment by woosung1223** [comment]
### 체크인
7점 - 어제 에어컨 온도조절에 실패해서 잠을 제대로 못잤읍니다.

### 체크아웃
7점 - 피곤해갖고 점수는 그대로임다.

**Comment by nlom0218** [comment]
### 체크인

8점 - 쏘쏘~~

### 체크아웃

9점 - 많이 해서 기분이 좋네요~~

**Comment by woo-jk** [comment]
### 체크인
8점 - 피곤하다 ㅜㅜ

### 체크아웃
7점 - 할게 많다 ㅜ

**Comment by MoonJeWoong** [comment]
### 체크인
9점 - 잠은 잘 못잤지만 프로젝트 마무리가 잘 되어 가는 것 같아서 기분은 좋네요~

### 체크아웃
10점 - 로그인 API 만들고 슬랙 로깅 알림도 구현해서 기분이 좋습니다~

**Comment by yeopto** [comment]
### 체크인

9점 - 푹 쉬다 와서 좋습니다.

### 체크아웃

7점 - 몇시간 풀 집중 했더니 컨디션 난조네용
